recipes/lsd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "lsds" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed" %}
```

Comment:
Doing the templating for the hash actually isn't considered best practice any more. Better to just leave the hash directly in the `sha256` key.

Reply:
My fault then. autotick bot seems to handle the situation fine, though? Is that because of the grayskull template?

Reply:
Yeah, the bot handles it fine, we just generally stick to templating name and version. Templating the hash doesn't have any real benefit.

Reply:
It makes the autotick bot diffs easier to read.

Reply:
And makes this recipe different than 90% of the others. 🤷‍♂️

Reply:
I have no strong opinions on the hash templating so I removed it for the sake of conforming with the norm. Good to go now?